doc/designs/subordinate-ids.md

```diff
@@ -76,21 +73,44 @@ basic use cases. Some restrictions may be lifted in the future.
 
 * subuid and subgids cannot be set independently. They are always set
   to the same value.
-* counts are hard-coded to value 65536
-* once assigned subids cannot be removed
+* counts are hard-coded to value 65536.
+* once assigned subids cannot be removed.
 * IPA does not support multiple subordinate id ranges, yet. Contrary to
   ``/etc/subuid``, users are limited to one set of subordinate ids. The
   limitation is implemented with a unique index on owner reference and
   can be lifted in the future.
-* subids are auto-assigned. Auto-assignment is currently emulated
-  until 389-DS has been extended to support DNA with step interval.
+* subids are auto-assigned from 389-DS DNA plugin (dnaInterval).
 * subids are allocated from hard-coded range
   ``[2147483648..4294901767]`` (``2^31`` to ``2^32-1-65536``), which
   is the upper 2.1 billion uids of ``uid_t`` (``uint32_t``). The range
   can hold little 32,767 subordinate id ranges.
 * Active Directory support is out of scope and may be provided in the
   future.
 
+
+Finally, it is important to note that new installations come with a new
+idrange for subordinate ids. To avoid backwards compatibility issues the
+new $REALM_subid_range entry includes base RIDs and a SID:
+
+```text
+dn: cn=IPASUBID.TEST_subid_range,cn=ranges,cn=etc,dc=ipasubid,dc=test
+objectClass: ipaDomainIDRange
+objectClass: ipaIDrange
+objectClass: ipaTrustedADDomainRange
+objectClass: top
+cn: IPASUBID.TEST_subid_range
+ipaBaseID: 2147483648
+ipaBaseRID: 2147083648
+ipaIDRangeSize: 2147352576
+ipaNTTrustedDomainSID: S-1-738065-838566-2378521612
+ipaRangeType: ipa-local-subid
+ipaSecondaryBaseRID: 2147283648
```

Comment:
In my IPA 4.9.9 instance, I see `ipaRangeType: ipa-ad-trust` (not ipa-local-subid).

Reply:
@flo-renaud @abbra, good catch, thanks, fixed :)